go/chat/boxer.go

```diff
@@ -52,7 +52,7 @@ type Boxer struct {
 	utils.DebugLabeler
 	globals.Contextified
 
-	boxWithVersion chat1.MessageBoxedVersion
```

Comment:
inside of `compareHeadersMBV2` can you change the equality check on the metadata to only happen if we're on V3? just a sanity check if we're only supporting the `EphemeralMetadata` on v3. (sorry this was leftover from my purge PR but i didn't have the v3 stuff)

https://github.com/keybase/client/pull/11420/files#diff-d1325d9a259c7d25092218f3fa0661d4R868